src/ldraw_to_scad/ldrawconverter.py

```diff
@@ -136,7 +133,7 @@ def make_function_name(name):
         """ Calculate OpenSCAD name from LDraw name. """
         function_name = name.lower().split('.', 1)[0]
         function_name = function_name.replace('\\', '__').replace('-', '_').\
-            replace('+', '_').replace(' ', '_')
+            replace('+', '_').replace(' ', '_').replace('#', '_')
```

Comment:
Can't help wonder if we need to use a python slugify type library for this? Otherwise we'll need to chase other special character types.

Reply:
Eventually, a smarter mechanism than chaining up the replace commands is likely better. Given that I translate the complete official LDraw parts library regularly and since recently also compile all parts to STL files automatically at least I should have caught everything that is in the library of more than 17,000 parts for now.